crates/toml_edit/src/parser/strings.rs

```diff
@@ -217,13 +218,15 @@ fn mlb_quotes<'i>(
     move |input: &mut Input<'i>| {
         let start = input.checkpoint();
         let res = terminated(b"\"\"", peek(term.by_ref()))
+            // Safety: ???
             .map(|b| unsafe { from_utf8_unchecked(b, "`bytes` out non-ASCII") })
```

Comment:
> t. However I'm unable to figure out why mlb_quotes and mll_quotes are safe. There is no call to bytes there, and it seems to be using a terminated parser.

`terminated` says to ignore the second parser, only returning the result of the first parser.  So in this case, we are effectively calling `from_utf8_unchecked("\"")`.  So the invariant we are upholding is that `"` is ASCII.